src/actions/index.js

```diff
@@ -9,10 +9,20 @@ export const DECREMENT = 'DECREMENT';
 // logic of updating the central Redux store. That
 // is left to the reducer(s).
 
-export const increment = () => {
+export const increment = count => {
   // Fill in this function
+  console.log('Increment Action Creator: ', count)
+  return {
+    type: INCREMENT,
+    payload: count
```

Comment:
No need for payload here

export const increment = () => {
  return { type: INCREMENT }
};